transactions-of-the-philological-society.csl

```diff
@@ -0,0 +1,221 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" initialize="false" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Transactions of the Philological Society</title>
+    <id>http://www.zotero.org/styles/transactions-of-the-philological-society</id>
+    <link href="http://www.zotero.org/styles/transactions-of-the-philological-society" rel="self"/>
+    <link href="https://onlinelibrary.wiley.com/page/journal/1467968x/homepage/forauthors.html" rel="documentation"/>
+    <author>
+      <name>Alexei Kassian</name>
+      <email>a.kassian@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="humanities"/>
+    <category field="linguistics"/>
+    <issn>0079-1636</issn>
+    <eissn>1467-968X</eissn>
+    <summary>TPhS</summary>
+    <updated>2025-01-31T15:24:24+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor" font-variant="normal" vertical-align="baseline">
```

Comment:
Remove ` font-variant="normal"` throughout your style unless it's be design (usually never is)